prow/genyaml/genyaml_test.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package genyaml
+
+import (
+	"testing"
+)
+
+func TestCommentGeneration(t *testing.T) {
+	SetPath("example_config.go")
+
+	expectedYaml := `# Plugin comment
```

Comment:
I very much recommend to not inline these but write them out into a file and have a flag to automatically update them, as described e.G. here: https://medium.com/soon-london/testing-with-golden-files-in-go-7fccc71c43d3

(I personally wouldn't name them `.golden` but just `.yaml`, but thats a detail)

Reply:
I like this pattern. I will update to use this pattern for the testing.